en/admin/index.rst

```diff
@@ -42,6 +42,10 @@ The service management utilities are as follows:
 
     *   :ref:`cubrid heartbeat <cubrid-heartbeat>`
 
+*   Java SP Server utility : Operates and manages the process of the Java stored procedure (Java SP) server.
```

Comment:
Operates --> operates 

(Capitalization After Colons: 
- When a colon introduces a list of of things, do not capitalize the first word after the colon unless it is a proper noun.
- When a colon introduces a phrase or an incomplete sentence that is meant to add information to the sentence before it, do not capitalize the first word after the colon unless it is a proper noun.

Therefore, there 'Operates' should change to 'operates', the lower case)